examples/dynamic-projections-dask.py

```diff
@@ -0,0 +1,60 @@
+"""
+Using dask array operations, one can dynamically take arbitrary slices
+and computations of a source dask array and display the results in napari.
+When the computation takes one or more parameters, one can tie a UI to
+them using magicgui.
+"""
+
+import napari
+from dask import array as da
+from dask.array.lib.stride_tricks import sliding_window_view
+from skimage import data
+
+##############################################################################
+# Part 1: using code to view a specific value.
+
+blobs = data.binary_blobs(length=64, n_dim=3)
+blobs_dask = da.from_array(blobs, chunks=(1, 64, 64))
+
+# original shape [60, 1, 1, 5, 64, 64], slice out singleton axes
+blobs_dask_windows = sliding_window_view(
+    blobs_dask, window_shape=(5, 64, 64)
+)[:, 0, 0, ...]  
+blobs_sum = da.sum(blobs_dask_windows, axis=1)
+viewer = napari.view_image(blobs_sum)
+
+napari.run()
```

Comment:
this run blocks the example, might be better to remove it?

Reply:
@alisterburt I just tried it having the two viewers open but because we currently have bugs with multiple viewers open (#2977), I'd rather have them open in sequence for now.